### PR TITLE
fix mixed deployments OCP 4.14 OCS 4.13

### PR DESCRIPTION
**Pull request by DanielOsypenko**
FIXES: https://github.com/red-hat-storage/ocs-ci/issues/8097

we need 4.14 locators to follow scenarios of the upgrade and scenarios with mixed versions
OCP 4.14, ODF 4.13, etc.

failures:
https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/465/12976/594644/594920/594921/log
https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/465/12976/594644/594922/594923/log

### CODE REVIEW

**Comment by DanielOsypenko** [comment]
> PR validation on existing cluster
> 
> Cluster Name: dosypenk-138-c Cluster Configuration: PR Test Suite: tier1 PR Test Path: tests/ui/test_validation_ui.py::TestUserInterfaceValidation::test_dashboard_validation_ui Additional Test Params: OCP VERSION: 4.14 OCS VERSION: 4.13 tested against branch: release-4.13
> 
> [Job UNSTABLE](https://url.corp.redhat.com/e15bd40) (some or all tests failed).

bz opened -> https://bugzilla.redhat.com/show_bug.cgi?id=2232101

**Review by ebondare — APPROVED** [review]

File: ocs_ci/ocs/ui/views.py [hunk 1]
```diff
@@ -33,6 +33,10 @@
     "ocp_page": f"Overview · {azure_managed}Red Hat OpenShift",
     "login_page_title": "Log in · Red Hat OpenShift",
 }
+# Bug opened in Jira https://issues.redhat.com/browse/OCPBUGS-15419. Tmp solution to check locators
```
Comment (am-agrawa) [inline ocs_ci/ocs/ui/views.py:36]: This jira is verified. Do we need to revert this change?

Comment (am-agrawa) [reply ocs_ci/ocs/ui/views.py:36]: Let's do it then

Comment (DanielOsypenko) [reply ocs_ci/ocs/ui/views.py]: > Let's do it then

fixed, thanks

**Review by am-agrawa — APPROVED** [review]

**Review by ebondare — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]
